README.md

```diff
@@ -1,4 +1,102 @@
-# housing-dashboard
+# SF Planning Housing Dashboard
```

Comment:
Maybe we call it Housing Production Dashboard (here and below) since thats what its called in the docs?

Reply:
Done